go.mod

```diff
@@ -17,6 +17,7 @@ require (
 	k8s.io/api v0.28.0
 	k8s.io/apimachinery v0.28.0
 	k8s.io/client-go v0.28.0
+	k8s.io/utils v0.0.0-20230505201702-9f6742963106
```

Comment:
Can we use `github.com/AlekSi/pointer` instead as it's already imported?
The `k8s.io.utils` is also not versioned.

Reply:
thanks for pointing that out! updated